plugin/storage/cassandra/schema/v004-go-tmpl-test.cql.tmpl

```diff
@@ -0,0 +1,43 @@
+-- There are total 4 queries here
```

Comment:
Instead of testing on all the queries and manually creating an expected result, I reduced the number of queries to ease the creation of expected result.

Reply:
that doesn't really make sense to me. We can either test that the template is correct or that the code using the template is correct. You're not doing the former by having another template, and the latter you can do with the primary template. We don't need to validate that the output of running template is "as expected", that's like testing the Go template package.

Reply:
**We don't need to validate that the output of running template is "as expected", that's like testing the Go template package.**
We are also removing comments and constructing individual query strings out of it by iterating over lines in the template output. The test checks this "individual query string" construction, which not just involves using the template package.

**and the latter you can do with the primary template.**
Sure, I think I can write an integration test for this.

Reply:
Added a -s flag to skip applying schema. It can be run using:

bash scripts/cassandra-integration-test.sh 4 v004 v2 -s

Reply:
Hi @yurishkuro , I have the integration test script ready. Should I merge it with the current `cassandra-intergration-test.sh`? This would require adding an additional 4th parameter `skip_apply_schema` for running the script